a/aether/aether-api-n-spi-ubi8.4.sh

```diff
@@ -0,0 +1,55 @@
+# ----------------------------------------------------------------------------
+#
+# Package       : aether-api, aether-spi
+# Version       : aether-1.13.1
+# Source repo   : https://github.com/sonatype/sonatype-aether
+# Tested on     : UBI: 8.4
+# Script License: Apache License 2.0
+# Maintainer's  : Sapana Khemkar <Sapana.Khemkar@ibm.com>
+#
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+#!/bin/bash
+set -e
+
+# Variables
+PACKAGE_NAME=sonatype-aether
+PACKAGE_URL=https://github.com/sonatype/sonatype-aether.git
+PACKAGE_VERSION=aether-1.13.1
+
+
+# install tools and dependent packages
+yum install -y git wget
+
+# install java
+yum -y install java-1.8.0-openjdk-devel
+
+#install maven
+cd /opt/
+wget https://www-eu.apache.org/dist/maven/maven-3/3.8.4/binaries/apache-maven-3.8.4-bin.tar.gz
+tar xzf apache-maven-3.8.4-bin.tar.gz
+ln -s apache-maven-3.8.4 maven
+export MVN_HOME=/opt/maven
+export PATH=${MVN_HOME}/bin:${PATH}
+mvn -version
+
+# Cloning the repository from remote to local
+cd /home
```

Comment:
Just a suggestion, can we use `$HOME` instead of `/home` for build and validating the package ?

Reply:
Default HOME is set to /root. So everything will go under /root folder. I think its better to keep under /home folder